.ec/policy.yaml

```diff
@@ -0,0 +1,33 @@
+#
+# If we need to diverge from the "official" policy for pre-merge EC checks,
+# the EC integration test scenario POLICY_CONFIGURATION param can be set to
+# github.com/enterprisecontract/ec-cli to use this file.
+#
+# Compare is to the official policy for Red Hat releases of EC here:
+# https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/config/stone-prd-rh01.pg1f.p1/product/EnterpriseContractPolicy/registry-rhtap-contract.yaml
+#
+description: 'Custom pre-merge policy for ec-cli'
+publicKey: 'k8s://openshift-pipelines/public-key'
+sources:
+  - name: Release Policies
+    data:
+      - github.com/release-engineering/rhtap-ec-policy//data
+      - oci::quay.io/konflux-ci/tekton-catalog/data-acceptable-bundles:latest
+    policy:
+      # Let's not pin the digest here
+      - oci::quay.io/enterprise-contract/ec-release-policy:latest
+    config:
+      include:
+        - '@redhat'
+      exclude:
+        # Not currently doing hermetic builds, see https://issues.redhat.com/browse/EC-360
+        - hermetic_build_task.*
```

Comment:
Actually, we [are doing hermetic builds](https://github.com/enterprise-contract/ec-cli/blob/78860f52c3d872531e12103309e3ff56619700f6/.tekton/cli-main-ci-push.yaml#L38-L39)